### ISSUE TITLE
Define PD Milestone per timeframe and competency

### ISSUE THREAD

**Issue by kfklein15** [CodeYourFuture/Programme#75]
**Background**
We have worked on identifying the levels per competencies on our [test board](https://github.com/users/kfklein15/projects/4/views/16?sliceBy%5Bvalue%5D=JS3)
_These will change since we have received feedback that we have too much mandatory/key coursework, but we'll keep this test board updated accordingly

**Acceptance criteria**
- Define min/max count of coursework per Level and per timeframe

**Owner**
@kfklein15 @LaraHuzjan 

**Stakeholder**
PD Syllabus

**Approver**
@SallyMcGrath

**Comment by kfklein15** [comment]
@SallyMcGrath ,
@LaraHuzjan and I started looking into the PD Milestone calculation using the above acceptance criteria we had discussed and would like your input on the following:
1. We don't think we can have a minimum of coursework. It's a small amount of coursework (especially once we remove many from mandatory/key to stretch, which is currently being done).
2. Timeframe: we did a first table looking at the modules, not competencies. But then we thought we wanted it in detail per competency and started with the second table. We think we need table no 2 to give us a better idea of where we are at, but before continuing to build it, we would like your insights.

If you prefer I can book in some time for us to go through it.
